### DISCUSSION TITLE
Can't connect to MS SQL Database

### DISCUSSION THREAD

**Discussion by MariusCostin** [dbt-msft/dbt-sqlserver#176] [Q&A] [answered]
Hello,

I am trying to connect to a MS SQL instance from my Linux machine and it seems I can't. When I run debug I get the following error:
![image](https://user-images.githubusercontent.com/84188720/140926197-8b316a11-c9d0-4bc6-86f0-03febc6286b9.png)

Also, the profiles.yml configuration is like this:
![image](https://user-images.githubusercontent.com/84188720/140926455-cefbbaab-cef5-4f6f-a86a-3a23cdb5baec.png)

Any ideas?

Thanks.
Marius

**Comment by dataders** [comment, marked as answer]
that issue is due to a change in dbt version 0.21.0. we don't have the official release out for dbt-sqlserver v0.21.0 yet so your options are:
1. `pip install dbt-sqlserver==0.20.1` (the latest 'official' release), or
2. `pip install dbt-sqlserver==0.21.0rc1` (the 'beta' release)

**Reply by MariusCostin** [reply]
Seems to be working now. Thanks a lot!
